core/directory.go

```diff
@@ -596,6 +599,15 @@ func (dir *Directory) Export(
 		defPB = dir.LLB
 	}
 
+	rec := progrock.RecorderFromContext(ctx)
+
+	vtx := rec.Vertex(
+		digest.Digest(identity.NewID()),
+		strings.Join([]string{"export directory", dir.Dir, "to host", destPath},
```

Comment:
nit: `fmt.Sprintf` would be more conventional here, but really splitting hairs

Reply:
You're right! I'm fixing that and merging